recipes-browser/cog/cog_0.12.0.bb

```diff
@@ -0,0 +1,17 @@
+require cog.inc
+require conf/include/devupstream.inc
```

Comment:
`require conf/include/devupstream.inc` has to be removed from the the cog_0.10.1.bb recipe (https://github.com/Igalia/meta-webkit/blob/main/recipes-browser/cog/cog_0.10.1.bb#L2)